base/src/main/java/io/spine/protobuf/AnyPacker.java

```diff
@@ -143,6 +144,47 @@ public static Message unpack(Any any) {
         return AnyPacker::unpackOrNull;
     }
 
+    /**
+     * Provides the function for unpacking messages of a given type from {@code Any}.
+     *
+     * <p>The function returns {@code null} for {@code null} input.
+     *
+     * <p>The function throws a {@link UnexpectedTypeException} if the actual type of the message
+     * does not match the given class.
+     *
+     * @param type
+     *         expected class of the messages
+     */
+    public static <T extends Message> Function<@Nullable Any, @Nullable T>
+    unpackFunc(Class<T> type) {
```

Comment:
I would go with `unpackFn`.

Reply:
We've already got an overload called `unpackFunc`. I'm leaving this as is for the sake of consistency.